Tests/SwiftDocCTests/Model/SemaToRenderNodeDictionaryDataTests.swift

```diff
@@ -194,7 +203,7 @@ class SemaToRenderNodeDictionaryDataTests: XCTestCase {
         XCTAssertEqual(genreProperty.name, "genre")
         XCTAssertTrue(genreProperty.readOnly ?? false)
         attributeTitles = genreProperty.attributes?.map{$0.title.lowercased()}.sorted() ?? []
-        XCTAssertEqual(attributeTitles, ["default value", "possible values"])
```

Comment:
question: Should this test verify that this information is displayed somewhere else on the page?

Reply:
Resolved b y tweaking the test.